cli/pkg/kctrl/cmd/app/init/constants.go

```diff
@@ -0,0 +1,17 @@
+// Copyright 2020 VMware, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package init
+
+const (
+	FetchContentAnnotationKey = "fetch-content-from"
+	LocalFetchAnnotationKey   = "kctrl.carvel.dev/local-fetch-0"
+)
+
+const (
+	FetchFromGithubRelease  string = "Github Release"
+	FetchManifestFromGit    string = "Git Repository"
+	FetchChartFromHelmRepo  string = "Helm Chart from Helm Repository"
+	FetchChartFromGit       string = "Helm Chart from Git Repository"
+	FetchFromLocalDirectory string = "Local Directory"
+)
```

Comment:
Seems like these would be better suited in `fetch.go`.
Also, maybe we can have a type for the different fetch options?

Reply:
As they are being used in vendir.go as well, hence not moving to fetch.go.